JavaScript-Runtime-(JSRT)-Reference.md

```diff
@@ -138,3 +139,20 @@ References of JSRT typedefs, constants, enumerations and APIs.
 * [[JsSetRuntimeMemoryLimit]]
 * [[JsStrictEquals]]
 * [[JsStringToPointer]]
+
+### New string API (Only non-Windows)
```

Comment:
Running "dumpbin /exports ChakraCore.dll" i can see these function.
So they are supported on windows. 
Sorry for this mistake.